### PR TITLE
chore(apicon): update word

### PR DESCRIPTION
**Pull request by xzf0587**
1. add generated info in the header.
2. add TODO in sample code.

### CODE REVIEW

**Review by blackchoey — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/headerComment.template [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// This file is auto generated by Teams Toolkit to provide you instructions and reference code to call your api.
```
Comment (blackchoey) [inline packages/fx-core/templates/plugins/resource/apiconnector/sample/headerComment.template:1]: @timngmsft can you also help review the wording here?

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/js/aad.template [hunk 2]
```diff
@@ -16,7 +16,7 @@ const teamsFx = new teamsfxSdk.TeamsFx(teamsfxSdk.IdentityType.App, {
 // Initializes a new axios instance to call {{config.APIName}} API.
 const appCredential = teamsFx.getCredential();
 const authProvider = new teamsfxSdk.BearerTokenAuthProvider(
-  // Please replace '<your-api-scope>' with actual api scope value.
+  // TODO: Please replace '<your-api-scope>' with actual api scope value.
```
Comment (blackchoey) [inline packages/fx-core/templates/plugins/resource/apiconnector/sample/js/aad.template:19]: @timngmsft please also help review the changes here. The purpose of this prefix is explicitly tell users they need to do something here.

**Review by MuyangAmigo — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/headerComment.template (hunk 1 above)
Comment (timngmsft) [reply packages/fx-core/templates/plugins/resource/apiconnector/sample/headerComment.template:1]: I would capitalize API.

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/js/aad.template (hunk 2 above)
Comment (timngmsft) [reply packages/fx-core/templates/plugins/resource/apiconnector/sample/js/aad.template:19]: TODO: Replace '<your-api-scope>' with your API scope value.

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/js/cert.template [hunk 3]
```diff
@@ -4,7 +4,7 @@ const teamsfxSdk = require("@microsoft/teamsfx");
 const teamsFx = new teamsfxSdk.TeamsFx();
 // Initializes a new axios instance to call {{config.APIName}} API.
 const authProvider = new teamsfxSdk.CertificateAuthProvider(
-  // Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
+  // TODO: Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
```
Comment (timngmsft) [inline packages/fx-core/templates/plugins/resource/apiconnector/sample/js/cert.template:7]: TODO: Add code to read your certificate and private key. Replace "<your-cert>" and "<your-private-key>" with your actual certificate and private key values.

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/ts/aad.template [hunk 4]
```diff
@@ -21,7 +21,7 @@ const teamsFx = new TeamsFx(IdentityType.App, {
 // Initializes a new axios instance to call {{config.APIName}} API.
 const appCredential = teamsFx.getCredential();
 const authProvider = new BearerTokenAuthProvider(
-  // Please replace '<your-api-scope>' with actual api scope value.
+  // TODO: Please replace '<your-api-scope>' with actual api scope value.
```
Comment (timngmsft) [inline packages/fx-core/templates/plugins/resource/apiconnector/sample/ts/aad.template:24]: TODO: Replace '<your-api-scope>' with your API scope value.

File: packages/fx-core/templates/plugins/resource/apiconnector/sample/ts/cert.template [hunk 5]
```diff
@@ -9,7 +9,7 @@ import {
 const teamsFx = new TeamsFx();
 // Initializes a new axios instance to call {{config.APIName}} API.
 const authProvider = new CertificateAuthProvider(
-  // Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
+  // TODO: Please add code to read your cert and private key, replace "<your-cert>" and "<your-private-key>" with actual value.
```
Comment (timngmsft) [inline packages/fx-core/templates/plugins/resource/apiconnector/sample/ts/cert.template:12]: (same as above)

File: packages/fx-core/tests/plugins/resource/apiconnector/expectedFiles/js/aad [hunk 6]
```diff
@@ -29,7 +31,7 @@ const teamsFx = new teamsfxSdk.TeamsFx(teamsfxSdk.IdentityType.App);
 // Initializes a new axios instance to call fake API.
 const appCredential = teamsFx.getCredential();
 const authProvider = new teamsfxSdk.BearerTokenAuthProvider(
-  // Please replace '<your-api-scope>' with actual api scope value.
+  // TODO: Please replace '<your-api-scope>' with actual api scope value.
```
Comment (timngmsft) [inline packages/fx-core/tests/plugins/resource/apiconnector/expectedFiles/js/aad:34]: Same as above

**Review by timngmsft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**